### ISSUE TITLE
Filter icons are missing labels

### ISSUE THREAD

**Issue by vincentmvdm** [dajuniors/phase5#28]
When used as filters in the sidebar, the icons are missing labels. This makes them hard to interpret and potentially inaccessible. Best is to have both an icon and a textual label, as well as an aria-label under the hood.

Steps to reproduce:

1. Navigate to the app and observe how the icons in the legend do have labels, while the icons in the filter section do not

**Comment by lbc546** [comment]
Added

**Comment by kathlu** [comment]
https://github.com/dajuniors/phase5/commit/e369f6c219a914a77cb3b9dde79000ca14e3e8be
